plugins/template-validator.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const boom = require('boom');
+const schema = require('screwdriver-data-schema');
+const templateSchema = schema.api.templateValidator;
+const validator = require('screwdriver-template-validator');
+
+/**
+ * Hapi Template Validator Plugin
+ *  - Validates sd-template.yaml and returns the parsed template with any
+ *    errors associated with it
+ * @method register
+ * @param  {Hapi.Server}    server
+ * @param  {Object}         options
+ * @param  {Function} next
+ */
+exports.register = (server, options, next) => {
```

Comment:
Need to register this in https://github.com/screwdriver-cd/screwdriver/blob/master/lib/registerPlugins.js#L12-L20